.gitignore

```diff
@@ -2,3 +2,5 @@
 .tox/
 *.pyc
 *.egg-info/
+
+.idea
```

Comment:
BTW, If you find yourself having to add `.idea` to `.gitignore` files because your IDE creates those files, you should try making a gitignore_global file, which will ignore those file for you locally across all repos: https://davidwalsh.name/global-gitignore